ndbench-core/src/main/java/com/netflix/ndbench/core/defaultimpl/NdBenchGuiceModule.java

```diff
@@ -42,9 +42,14 @@
     protected void configure()
     {
         bind(NdBenchMonitor.class).to(FakeMonitor.class);
-        //bind(IClusterDiscovery.class).to(AWSLocalClusterDiscovery.class);
-        bind(IClusterDiscovery.class).to(LocalClusterDiscovery.class);
-//        bind(IClusterDiscovery.class).to(CfClusterDiscovery.class);
+        String discoveryEnv = System.getenv(NdBenchConstants.DISCOVERY_ENV);
```

Comment:
Can you update the wiki on how to use/ configure ClusterDiscovery in general not just specific to Cloud Foundry? 

Wiki - https://github.com/Netflix/ndbench/wiki/Configuration#default-implementation